javascript/javascript2/week1/Name_Generator.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+    <header>
+        <title>Spirit Animal Name Generator</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    </header>
+
+    <body>
+        <!--Create an input element, a button and a tag to display the spirit animal into-->
+        
+            <label for="name">Enter your name</label>
+            <input type="text" id="name">
+            <button id="submit">Your Spirit Animal</button>
+
+        
+        
+        <h2 id="animal"></h2>
+
+
+        <script type="text/javascript">
+
+        "use strict";
+
+            const button_element = document.getElementById("submit");
+            
+            const h2_element = document.getElementById("animal")
+            
+            const animalArray = ["old wolf", "gray eagle", "fast gazel", "crying butterfly", "white cat", "roaring lion", "brown bear"];
+           
+            button_element.addEventListener('click', nameGenerator);
+
+            const body_element = document.querySelector('body');
+            
+            const p_element = document.createElement('p');
+
+            function nameGenerator() 
+            {
+                const input_value = document.getElementById("name").value;
```

Comment:
```suggestion
                const name= document.getElementById("name").value;
```
Try and describe what a variable contains not what it is